crafting_templates.xml

```diff
@@ -1682,78 +1682,6 @@
       <UsablePiece piece_id="crpg_wide_leaf_spear_t4_handle" />
       <UsablePiece piece_id="crpg_wide_leaf_spear_t4_pommel" />
     </UsablePieces>
-  </CraftingTemplate>
-    <CraftingTemplate id="crpg_Spear4DMode" item_type="Polearm" modifier_group="polearm" item_holsters="polearm_back:polearm_back_2:polearm_back_3:polearm_back_4" default_item_holster_position_offset="0,0,-0.30" use_weapon_as_holster_mesh="true">

```

Comment:
this branch need to be rebased. If you don't know how to do it , you can also back up your xmls , pull main, create a new branch from main , then put back crafting_piece.xml since you only made change to crafting piece. i've seen some error in crafting piece. Since you will have to do changes in weapon.xml, i highly advise you rebase / recreate your branch before touching weapon.xml